portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/Pagination.path

```diff
@@ -42,7 +42,7 @@
 </tr>
 <tr>
 	<td>NEXT_LINK</td>
-	<td>//ul[contains(@class,'pagination')]/li/a[contains(.,'Next')] | //ul[contains(@class,'pagination')]/li[@class='']/a/span[contains(@class,'icon-caret-right')] | //a[contains(@class, 'page-link')]//*[contains(@class,'right')]</td>
+	<td>//ul[contains(@class,'pagination')]/li/a[contains(.,'Next')] | //ul[contains(@class,'pagination')]/li[@class='']/a/span[contains(@class,'icon-caret-right')] | //a[contains(@class, 'page-link')]//*[contains(@class,'right')] | //*[contains(@class, 'right') and contains(@focusable, 'false')]</td>
```

Comment:
Since a [condition was added in the removePermissionPG macro](https://github.com/brianchandotcom/liferay-portal/pull/137250/files#diff-ed9f7dc1b1ce1edcfcab8017fd7e72f73721d8b8056f9470dfdbb0a15f2dfae7R260) to expand items per page, the IsEditable function fails because the NEXT_LINK path does not find the "next link" if it is disabled.

As a solution I added an "or" to find the "next link" when it's disable